lib/browser/browserIndex.js

```diff
@@ -11,7 +11,5 @@
  *
  */
 
-var fetchRequest = require('../fetch/fetchRequest');
-var storageUtil = require('./browserStorage');
-
-module.exports = require('./browser')(storageUtil, fetchRequest);
+import OktaAuthBrowser from './browser';
```

Comment:
The naming convention here bothered me a little bit, as it's already in `browser` folder, why not just use `browser/index`?

Reply:
no real argument, it was like that when I found it

Reply:
I know, as we are making the change, not sure if now it's a good time to make the change? I would prefer simple `index` file, since it can make import from other modules easier. It's definitely fine to keep it unchanged.

Reply:
i did make the change